src/main/java/io/anserini/analysis/MegaTokenizer.java

```diff
@@ -0,0 +1,97 @@
+/*
+ * Anserini: A Lucene toolkit for reproducible information retrieval research
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.anserini.analysis;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.apache.lucene.analysis.Analyzer;
+import org.apache.lucene.analysis.core.WhitespaceAnalyzer;
+
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.Map;
+
+public class MegaTokenizer {
+  private static final Logger LOG = LogManager.getLogger(CompositeTokenizer.class);
+
+  private static final Map<String, String> tokenizerMap = new HashMap<>() {
+    {
+      put("ar", "theyorubayesian/bert-base-uncased-arabic-cc");
```

Comment:
@lintool I'll need an invite to the HuggingFace organization so I can push these tokenizers there.